microsoft-edge/progressive-web-apps-chromium/how-to/index.md

```diff
@@ -1,113 +1,139 @@
-       <link rel="shortcut icon" href="https://c.s-microsoft.com/favicon.ico?v2" />
-       <title>My Sample PWA</title>
-     </head>
-     <body>
-       <h1>My Sample PWA</h1>
-     </body>
-   </html>
-   ```
+Your front-end code uses HTML, CSS, JavaScript, and a JSON manifest only.
 
-1. Start a local web server by using the `http-server` Node.js library:
+You use HTML to describe the content in your app, such as the text, images, text fields, or buttons that appear in the user interface. You then use CSS to organize the HTML content in a layout, and provide styles to elements. You use JavaScript to add user interactions to your user interface. And finally, you use a JSON manifest file that describes your application to the host operating system.
 
-   ```shell
-   cd path\to\MySamplePWA
-   npx http-server
-   ```
-  
-   In the above commands, replace `path\to\MySamplePWA` with the actual path where you create the `MySamplePWA` folder.
+Note that although your front-end code runs by using the device's web browser, the browser user interface may not be visible as your app can choose to run in a standalone window.
 
-You now have a simple, functional website running on a local web server.
+On top of the user interface code, you also use JavaScript to make your application faster, more reliable, and network-independent by using a service worker file. Finally, your front-end code also contains a JSON manifest file that describes your application to the host operating system.
 
-Now browse to `http://localhost:8080` to view your new website.
+Below is a diagram of the high-level architecture of a PWA:
 
-![Running your new PWA on localhost.](../media/sample-pwa-app.png)
+![Architecture diagram of a PWA, showing the web server on one side, and the device on the other. The device part contains the front-end code with HTML, CSS, JavaScript, the service worker, and the manifest](../media/pwa-architecture.png)
 
 
 <!-- ====================================================================== -->
-## Getting started building a PWA
+## Step 1 - Start a web server
 
-Now that you have a simple website, extend it as a Progressive Web App (PWA) by adding the three requirements for PWAs:
+PWAs are distributed to users by using web servers. Once your app is ready, deploy it to the web by using a web hosting provider. You can then update your app simply by deploying the new version to your web server again.
 
-*  [HTTPS](#step-1---use-https).
-*  A [Web App Manifest](#step-2---create-a-web-app-manifest).
-*  A [Service Worker](#step-3---add-a-service-worker).
+To start developing your PWA, you can use a local web server instead. To start a local server:
 
+1. Create a new folder on your computer where the web server will run.
 
-<!-- ====================================================================== -->
-## Step 1 - Use HTTPS
+    You can do this by opening a command prompt and typing:
+    
+    ```shell
+    cd path\to\your\dev\folder
```

Comment:
are backslashes ok for git bash on Windows or for Mac or Linux?

Reply:
I'm changing them to forward slashes. I think `/` are more common in technical docs and will work on mac, linux, and git bash on windows. Users of the Windows command prompt likely already know that they need to switch them to `\` instead.